internal/schema/schema_test.go

```diff
@@ -0,0 +1,370 @@
+package schema
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"errors"
+	enginev1 "github.com/cerbos/cerbos/api/genpb/cerbos/engine/v1"
+	schemav1 "github.com/cerbos/cerbos/api/genpb/cerbos/schema/v1"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/structpb"
+	"testing"
+)
+
+func TestValidate(t *testing.T) {
+	t.Run("schema.Manager.Validate", func(t *testing.T) {
+		var testCases = []struct {
+			name    string
+			input   enginev1.CheckInput
+			wantErr bool
+		}{
+			{
+				name: "Test 1",
+				input: enginev1.CheckInput{
```

Comment:
You could have used similar logic to [engine tests](https://github.com/cerbos/cerbos/blob/2a660b51e837a22f7c9f31a36e1b9cb0f16ebe53/internal/engine/engine_test.go#L57) (and most other tests in this repo) to read these from YAML files -- which are much easier to construct than writing protobufs by hand.